### PR TITLE
Configure mtq jobs

### PR DESCRIPTION
**Pull request by Barakmor1**
Add presubmit and postsubmit jobs to mtq operator
/cc @brianmcarey

### CODE REVIEW

**Comment by brianmcarey** [comment]
@Barakmor1 are the 1.1 lanes expected to pass? 

/retest-required

**Comment by Barakmor1** [comment]
@brianmcarey I want to test post submit jobs before back-porting to 1.1 is that possible before testing post-submit before merging this one?

**Comment by Barakmor1** [comment]
@brianmcarey i'm changing the jobs to presubmit temporary so i could test it

**Comment by brianmcarey** [comment]
> @brianmcarey I want to test post submit jobs before back-porting to 1.1 ,is that possible testing post-submit before merging this one?

Unfortunately we don't have rehearsals available for postsubmits as we don't want rehearsal jobs actually publishing artifacts.

**Comment by Barakmor1** [comment]
> Unfortunately we don't have rehearsals available for postsubmits as we don't want rehearsal jobs actually publishing artifacts.

i'm changing the jobs to presubmit temporary so i could test it

File: github/ci/prow-deploy/files/jobs/kubevirt/managed-tenant-quota/managed-tenant-quota-presubmits-1.1.yaml [hunk 1]
```diff
@@ -0,0 +1,126 @@
...
+      - release-v1.1
+    cluster: ibm-prow-jobs
+    always_run: true
+    optional: false
+    decorate: true
+    decoration_config:
+      timeout: 1h
+      grace_period: 5m
+    labels:
+      preset-dind-enabled: "true"
```
Comment (brianmcarey) [inline github/ci/prow-deploy/files/jobs/kubevirt/managed-tenant-quota/managed-tenant-quota-presubmits-1.1.yaml:15]: Can you update this to the podman preset?

File: github/ci/prow-deploy/files/jobs/kubevirt/managed-tenant-quota/managed-tenant-quota-presubmits-1.1.yaml [hunk 2]
```diff
@@ -0,0 +1,126 @@
...
+      - release-v1.1
+    cluster: ibm-prow-jobs
+    always_run: true
+    optional: true
+    decorate: true
+    decoration_config:
+      timeout: 1h
+      grace_period: 5m
+    labels:
+      preset-dind-enabled: "true"
```
Comment (brianmcarey) [inline github/ci/prow-deploy/files/jobs/kubevirt/managed-tenant-quota/managed-tenant-quota-presubmits-1.1.yaml:43]: Same here - is there a reason using for older dind preset?

File: github/ci/prow-deploy/files/jobs/kubevirt/managed-tenant-quota/managed-tenant-quota-presubmits-1.1.yaml [hunk 3]
```diff
@@ -0,0 +1,126 @@
...
+    always_run: true
+    optional: false
+    decorate: true
+    decoration_config:
+      timeout: 5h
+      grace_period: 5m
+    max_concurrency: 6
+    cluster: kubevirt-prow-workloads
+    labels:
+      preset-dind-enabled: "true"
```
Comment (brianmcarey) [inline github/ci/prow-deploy/files/jobs/kubevirt/managed-tenant-quota/managed-tenant-quota-presubmits-1.1.yaml:105]: please try with the podman preset

**Review by brianmcarey — COMMENTED** [review]
Looks good - just a couple of comments on the presets

File: github/ci/prow-deploy/files/jobs/kubevirt/managed-tenant-quota/managed-tenant-quota-presubmits-1.1.yaml (hunk 1 above)
Comment (Barakmor1) [reply github/ci/prow-deploy/files/jobs/kubevirt/managed-tenant-quota/managed-tenant-quota-presubmits-1.1.yaml]: Done

File: github/ci/prow-deploy/files/jobs/kubevirt/managed-tenant-quota/managed-tenant-quota-presubmits-1.1.yaml (hunk 2 above)
Comment (Barakmor1) [reply github/ci/prow-deploy/files/jobs/kubevirt/managed-tenant-quota/managed-tenant-quota-presubmits-1.1.yaml]: Done

File: github/ci/prow-deploy/files/jobs/kubevirt/managed-tenant-quota/managed-tenant-quota-presubmits-1.1.yaml (hunk 3 above)
Comment (Barakmor1) [reply github/ci/prow-deploy/files/jobs/kubevirt/managed-tenant-quota/managed-tenant-quota-presubmits-1.1.yaml]: Done

**Pull request merged**